ur_controllers/CMakeLists.txt

```diff
@@ -1,35 +1,52 @@
 cmake_minimum_required(VERSION 3.5)
 project(ur_controllers)
 
-add_compile_options(-std=c++11)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 14)
```

Comment:
The old file used two spaces for indentation. let's keep this...

```suggestion
  set(CMAKE_CXX_STANDARD 14)
```

Reply:
Please use the .clang-format file in the root of the project